ctest.h

```diff
@@ -391,7 +419,7 @@ static void color_print(const char* color, const char* text) {
         printf("%s\n", text);
 }
 
-#ifdef __APPLE__
+#if defined (__APPLE__) && !defined (_POSIX_C_SOURCE)
```

Comment:
setup/teardown is definately not an Apple-only feature! However, for Apple we need to do an extra step, the find_symbol() thing..